docs: added scaffolding, config, proto and frontend (#891)
tendermint/starport: docs/5 IBC and Relayer/1 Relayer.md

# Relayer

Starport has a built-in IBC relayer that let's you connect local (running on you local machine) and remote blockchains. Starport relayer uses [IBC Go relayer](https://github.com/cosmos/relayer).

## `starport relayer configure`

> `configure` command configures a connection between two blockchains. You will be prompted for RPC endpoints (required) and faucet endpoints (optional). Accounts used by the relayer are created on both blockchains and faucets are used (if available) to automatically fetch tokens. If the relayer fails to recieve tokens from a faucet, you will need to manually send tokens to addresses. By default a connection is setup for the `ibc-transfer` module for token transfers.

Reviewer: ```suggestion
The `configure` command configures a connection between two blockchains. You are prompted for the required RPC endpoints and optional faucet endpoints. Accounts used by the relayer are created on both blockchains and faucets are used, if available, to automatically fetch tokens. If the relayer fails to receive tokens from a faucet, you must manually send tokens to addresses. By default, a connection for token transfers is set up for the `ibc-transfer` module.
```

> Optional `--advanced` flag: let's you configure port and version for custom IBC module.
> 
> By default configuration is stored in `$HOME/.relayer/`.

Reviewer: ```suggestion
The optional `--advanced` flag lets you configure port and version for the custom IBC module.

By default, relayer configuration is stored in `$HOME/.relayer/`.
```

## Example

> All values can be passed with flags.

Reviewer: ```suggestion
All flags are optional. All values can be passed with flags.
```

Reviewer: I am the type of learner who enjoys a narrative that goes along with a technical command.

Author: Yes, I'll add description in a separate PR.
